libraries/AP_HAL/hwdef/scripts/hwdef.py

```diff
@@ -301,6 +302,7 @@ def parse_i2c_device(self, dev):
         elif a[1] == 'ALL':
             return ('FOREACH_I2C(b)', 'GET_I2C_DEVICE(b,0x%02x)' % (busaddr))
         busnum = int(a[1])
```

Comment:
We need some comments in here stating very clearly that i2c_device_list is incomplete - i.e. doesn't contain all internal i2c devices.